### PR TITLE
Release v0.12.

### PR DESCRIPTION
**Pull request by fivetran-avinash**
**Are you a current Fivetran customer?** 

**What change(s) does this PR introduce?** 

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- Fixed `jira__daily_issue_field_history` model to make sure component values are correctly joined into our issue models ([#81](https://github.com/fivetran/dbt_jira/pull/81)).
- Please note, a `dbt run --full-refresh` will be required after upgrading to this version in order to capture the updates.
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#80] & [#81]
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [x] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🪁 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ vars:
     jira_user_identifier: "user"
     jira_version_identifier: "version"
 
-    issue_field_history_columns: ['summary', 'components', 'story points']
+  issue_field_history_columns: ['summary', 'components', 'story points']
```
Comment (fivetran-reneeli) [inline integration_tests/dbt_project.yml:32]: @fivetran-avinash is this made into a global variable? In that case I think we should prefix it with "jira__"

**Review by fivetran-reneeli — CHANGES_REQUESTED** [review]
@fivetran-avinash  looks good except just 1 comment I had about global variable naming!

**Comment by fivetran-joemarkiewicz** [comment]
> @fivetran-avinash looks good except just 1 comment I had about global variable naming!

@fivetran-reneeli @fivetran-avinash this late in the development cycle I would request to keep the variable name as it is. That variable is used extensively throughout this package and renaming it would cause quite a deal of effort. May we possibly open an FR for a future use case and keep it as is for the time being.

**Review by fivetran-reneeli — APPROVED** [review]
Looks good!

**Pull request merged**